README.md

```diff
@@ -1,68 +1,44 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Quiz web app
 
-## Available Scripts
+## Description:
+This is a quiz web app which helps users to check their knowledge of CYF study material.
 
-In the project directory, you can run:
+## Installation: 
+1. Clone files into your local machine using - `git clone https://github.com/CodeYourFuture/project-quiz.git`.
+2. Run `npm install` in your local terminal.
+3. Run `npm start` to run the app or open `http://localhost:3000` to view it in the browser.
 
-### `yarn start`
+## Table of Quizzes: 
+    1.HTML.
+    2.CSS.
+    3.JavaScript.
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Running the tests:
+    TBC.
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+## Tech Used / Dependencies:
+ React.
+ HTML.
+ CSS.
+ Node.
```

Comment:
```suggestion
- React
- HTML
- CSS
```

It looks like you are listing things, so let's use a list. Also, we are not using Node right now so we shouldn't mention it yet

Reply:
It looks like you forgot to remove the indentation — let's remove it.

Here, we are not listing something that has a specific order, so we should use bullet points. Also, with lists we should not have periods at the end if they are not complete sentences.